andela/travel_tool_front
# #160663195 Fix Approve and Reject Button Color

#### What does this PR do?
Fix approval and reject button color when clicked

#### Description of Task to be completed?
- Fix approve button color when clicked
- Fix reject button colour when clicked
- Fix modal width when notification is clicked

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- Run the server `make start`
- Navigate to localhost:3000
- Login as a manager role using your andela account
- Navigate to `My approval` page
- Click on any travel requests to be approved or rejected.
- Click approve or reject button
- The appropriate color displays
- Exit the modal
- Click notification on the notification pane, it should be of the same size with the request modal

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160663195](https://www.pivotaltracker.com/story/show/160663195)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-20 at 8 59 47 pm" src="https://user-images.githubusercontent.com/28366272/45844094-4d173a00-bd19-11e8-84e0-64c010ee9f87.png">
<img width="1440" alt="screen shot 2018-09-20 at 8 59 56 pm" src="https://user-images.githubusercontent.com/28366272/45844095-4d173a00-bd19-11e8-9e96-fb83e8659dca.png">
<img width="1440" alt="screen shot 2018-09-20 at 9 22 02 pm" src="https://user-images.githubusercontent.com/28366272/45844741-3e318700-bd1b-11e8-8d9b-effdd1b439a3.png">


#### Questions:
N/A
